Source/ThirdParty/libwebrtc/Source/webrtc/common_video/h265/h265_sps_parser.cc

```diff
@@ -128,6 +128,7 @@ absl::optional<H265SpsParser::SpsState> H265SpsParser::ParseSpsUpToVui(Bitstream
   }
   // sps_seq_parameter_set_id: ue(v)
   golomb_ignored = buffer->ReadExponentialGolomb();
+  (void)golomb_ignored; // Ignore -Wunused-but-set-variable warning.
```

Comment:
Another approach to fixing this would be to remove the `golomb_ignored` variable and to do this instead:

```
  (void)buffer->ReadExponentialGolomb();
```

I wasn't sure what was preferred.